README.md

```diff
@@ -48,7 +48,7 @@ Replace `<machine-name>` with the machine you want to connect to (from `info` ca
 
 ### Building AMIs
 
-    $ ./CardanoCSL.hs ami
+    $ ./CardanoCSL.hs ami -c config.yaml
```

Comment:
You're supposed to craft one. As shown here: https://github.com/input-output-hk/iohk-nixops#cardano-explorer

I will revise this as part of DEVOPS-29